target/linux/lantiq/image/ar9.mk

```diff
@@ -119,6 +119,8 @@ define Device/netgear_dgn3500b
   DEVICE_MODEL := DGN3500B
   SOC := ar9
   IMAGE_SIZE := 16000k
+  KERNEL := kernel-bin | append-dtb | lzma | loader-kernel | uImage none
+  KERNEL_INITRAMFS := $$(KERNEL)
```

Comment:
I think this fix can also be applied to DGN3500. Please add the same things under `
define Device/netgear_dgn3500`.  As for `KERNEL_ INITRAMFS:=$$(KERNEL)`, I am not sure if it is necessary. It only affects initramfs. In any case, it won't hurt.

It would be better to add some more information to commit message such as fix lzma decompress issue by using lzma loader. You can use `git commit --amend` to change commit.